### PR TITLE
Remove unnused /scripts/ChangelogConfig.grovy file

### PR DESCRIPTION
**Pull request by clockworkorange**
Issue: https://github.com/docToolchain/docToolchain/issues/462

### All Submissions:

* [x] Does your PR affect the documentation? ✅
* [x] If yes, did you update the documentation or create an issue for updating it? ✅

The source of the documentation can be found in `/src/docs/manual`.
To "publish" it, execute `./gradlew exportContributors && ./gradlew && ./copyDocs.sh`.
This will convert the `.adoc` file to HTML and copy them to the right folder so that github pages will pick them up.

If you didn't find the time to update docs, please create an issue as reminder to do so.

### Your first submission

* [ ] Welcome to the list of contributors! If you have any questions, feel free to ask them by creating a new issue
* [ ] Have you added your name to the list of [contributors.adoc](https://github.com/docToolchain/docToolchain/blob/master/src/docs/manual/05_contributors.adoc)? 
No.
I would like to appear in the contributors list

### New Feature Submissions:

1. [ ] Did you create new tests for your submission? ❌
2. [x] Does your submission pass all tests? (see travis check) ✅

### Changes to Core Features:

* [ ] Have you written new tests for your core changes, as applicable? ❌
* [x] Have you successfully ran tests with your changes locally? ✅

### Change to Documentation:

* [x] Did you build the docs and copy them to the `/docs` folder? ✅
* [ ] If not, did you create an issue for doing so?

inspiration: https://github.com/stevemao/github-issue-templates

### CODE REVIEW

File: docs/htmlchecks/index.html [hunk 1]
```diff
@@ -104,16 +104,16 @@ <h2>Results by Page </h2>
...
+        <td class="number">689</td>
+        <td class="number">3</td>
         <td class="number failures">99%</td>
       </tr>
 
     </tbody>
   </table><hr>
 
 <h1 id="manualXhtml">Results for manual.html </h1>
-location : /workspace/docToolchain/build/docs/html5/manual.html <p>
```
Comment (ewoks) [inline docs/htmlchecks/index.html:116]: @clockworkorange I'm pretty sure you didn't want to change this one

Comment (clockworkorange) [reply docs/htmlchecks/index.html]: Sorry. Fixed

**Comment by rdmueller** [comment]
thanx for this PR! I will resolve the conflicts asap...

**Comment by rdmueller** [comment]
have to admit that fixing this conflict seems to be harder for me than I thought...

**Comment by rdmueller** [comment]
Ha! Managed to merge it :-) 
Thanx for this PR!!!! I learned a lot!

**Pull request merged**